Dockerfile

```diff
@@ -16,10 +16,21 @@ RUN cd ./backend && go build -o ./server ./cmd/
 
 RUN cd ./frontend && npm install && npm run build
 
-# Create a plugins folder if none exists so we copy whatever exists to the
-# image.
+# Backwards compatibility, move plugin folder to only copy matching plugins.
+RUN mv plugins plugins-old
+
 RUN mkdir -p ./plugins
 
+# Backwards compatibility, copy any plugins found inside "./plugins" into ".plugins".
+# They should match plugins/MyFolder/main.js, otherwise they are not copied.
+RUN find plugins-old -maxdepth 2 -type f  | grep -i main.js$ | xargs -i dirname {} | xargs -i cp -a {} ./plugins/
+
+RUN find .plugins -maxdepth 2 -type f  | grep -i main.js$ | xargs -i dirname {} | xargs -i cp -a {} ./plugins/
```

Comment:
Should we be a bit more strict here and copy the files only if they are in `.plugins/*/main.js`? (i.e. why do we allow a depth of 2? Are we expecting `.plugins/main.js`?)

Reply:
I added a mindepth 2 so it only matches plugins/MyFolder/main.js